lib/greed/score.rb

```diff
@@ -0,0 +1,26 @@
+module Greed
+  module Score
+    def calcScore(dice)
+      raise ArgumentError.new unless dice
+      score = 0
+
+      # Create hash of count of each die
+      counts = dice.inject(Hash.new(0)) {|hash, num| hash[num] += 1; hash}
+
+      # Calculate score
```

Comment:
You've broken the scoring into steps, did you consider separating those jobs into their own methods?